user/forms.py

```diff
@@ -0,0 +1,24 @@
+"""
+User application forms
+"""
+
+from django.contrib.auth import forms
+from user.models import UserModel
+
+
+class RegisterForm(forms.UserCreationForm):
+    class Meta:
+        model = UserModel
+        fields = ["username", "first_name", "last_name", "email", "age", "sex", "password1", "password2", ]
```

Comment:
Ось таке імʼя для цього поля буде більш правильним
```suggestion
        fields = ["username", "first_name", "last_name", "email", "age", "gender", "password1", "password2", ]
```